.github/workflows/lock.yml

```diff
@@ -13,6 +13,7 @@ concurrency:
 
 jobs:
   action:
+    if: ${{ github.repository_owner == 'videojs' }}
```

Comment:
This will cause the jobs to be skipped on forks. They'll still show in the actions list.

Reply:
@mister-ben thank you, that's even better !